src/utilities/database_utils.ts

```diff
@@ -487,3 +504,41 @@ export async function removeProject(
         return [response, []]
     }
 }
+
+export async function getLastModifiedInstaller(
+    db: PouchDB.Database<Base>,
+): Promise<Installer | null> {
+    await db.info()
+    const findRequest: PouchDB.Find.FindRequest<Base> = {
+        selector: {
+            type: { $eq: 'project' },
+            'metadata_.last_modified_at': { $ne: '' },
+            'data_.installer.company_name': { $ne: '' },
+        },
+        fields: ['data_.installer', 'metadata_.last_modified_at'],
+    }
+
+    const findResponse: PouchDB.Find.FindResponse<any> =
+        await db.find(findRequest)
+
+    if (findResponse.docs.length > 0) {
+        const lastModifiedProject = findResponse.docs.sort((a, b) => {
```

Comment:
Instead of manually sorting `findResponse.docs` and then selecting the first document, use PouchDB [`db.find(sort: { 'metadata_.last_modified_at': 'desc', limit: 1 })`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pouchdb-find/index.d.ts#L84).